content/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -10,17 +10,19 @@ workarounds that can help avoid these potential problems:
 Many of the DVC commands have POSIX-style options that are given with a double
 dash `--`. This isn't supported by the simple Windows command prompt `cmd`.
 
-Common Windows terminal alternatives are
-[Git Bash](https://gitforwindows.org/#bash) or
-[Anaconda Prompt](https://docs.anaconda.com/anaconda/user-guide/getting-started/#open-prompt-win)
-– but they may not support all the POSIX features (e.g. `\` line continuation).
-
 💡 We recommend the full [Cmder](https://cmder.net/) console emulator (which
 already includes _Git for Windows_).
 
-Its also possible to enjoy a full Linux terminal experience with the
-[WSL](https://blogs.windows.com/windowsdeveloper/2016/03/30/run-bash-on-ubuntu-on-windows/)
-– but it may not be possible to access GPUs from this subsystem.
+Other common terminal alternatives for Windows are
```

Comment:
still quite complicated, we have three paragraphs now cmd, git bash and anaconda, then a note with WSL ... it should be as simple as - cmd is bad, try to use environments like CmdEr, Windows Terminal + WSL, etc.

Reply:
OK Ivan I pushed a new version. It's clearer but not shorter. What do you think? I've actually been trying these options (except Anaconda Prompt) all this time which is why I keep adding caveats.

Reply:
looks good to me, except the last note about git problems on win

Reply:
OK I removed the whole note.

Reply:
👍